### ISSUE TITLE
[Noetic] rtsprofile fails to build on all platforms

### ISSUE THREAD

**Issue by sloretz** [tork-a/rtsprofile-release#1]
* https://build.ros.org/view/Nbin_uF64/job/Nbin_uF64__rtsprofile__ubuntu_focal_amd64__binary/95/
* https://build.ros.org/view/Nbin_ufhf_uFhf/job/Nbin_ufhf_uFhf__rtsprofile__ubuntu_focal_armhf__binary/95/
* https://build.ros.org/view/Nbin_ufv8_uFv8/job/Nbin_ufv8_uFv8__rtsprofile__ubuntu_focal_arm64__binary/95/

```
23:05:08 make[4]: Entering directory '/tmp/binarydeb/ros-noetic-rtsprofile-2.0.0/.obj-x86_64-linux-gnu'
23:05:08 cd /tmp/binarydeb/ros-noetic-rtsprofile-2.0.0 && python setup.py build
23:05:08 /bin/sh: 1: python: not found
```

Opening here as https://github.com/gbiggs/rtsprofile seems unmaintained.

Seems similar to https://github.com/tork-a/rtctree-release/issues/3

**Comment by k-okada** [comment]
@sloretz sorry for trouble, fixed in ros/rosdistro#39464

**Comment by k-okada** [comment]
closed via ros/rosdistro#39464
